ifrs17/Constants/Validations.ipynb

```diff
@@ -45,7 +45,7 @@
             "outputs": []
         },
         {
-            "cell_type": "code",
+            "cell_type": "markdown",
```

Comment:
this is wrong. It looks like a merge conflict.

Reply:
Yes, absolutely right. To be on the safe side, I redid now the merger from the scratch. I think it is correct now